packages/sdk/src/build/build-arguments.ts

```diff
@@ -15,6 +15,6 @@ export function args(ax = []) {
  * @param {Function} xform - A function to transform the value
  * @returns {object} - An argument object
  */
-export function arg(value, xform) {
+export function arg(value: any, xform: any) {
```

Comment:
```suggestion
export function arg<T extends TypeDescriptor<any>>(value: TypeDescriptorInput<T>, xform: T) {
```

Can we strongly type this using the types from `@onflow/types`.  I think the suggestion above is how it would look - not 100% sure that TypeDescriptorInput is exported, you may have to export it from the package if not.